dvc/repo/experiments/save.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import os
+from typing import TYPE_CHECKING, List, Optional
+
+from funcy import first
+
+if TYPE_CHECKING:
+    from dvc.repo import Repo
+
+
+logger = logging.getLogger(__name__)
+
+
+def save(
+    repo: "Repo",
+    name: Optional[str] = None,
+    force: bool = False,
+    include_untracked: Optional[List[str]] = None,
```

Comment:
> > 2\. Neither `dvc exp run` or `dvc exp save` saves arbitrary new files, even if they are added to git. This is related to the idea from the previous PR that it should include all untracked files. It might not be a blocker for now if solving the first issue is enough for dvclive, but it seems like there needs to be some mechanism to save new files to an experiment (either all untracked files or at least those that have been added to git).

@dberenbaum To cover the DVCLive scenario I included this option (which is not exposed in the CLI but I can expose it).

This allows to pass pattern(s) of (potentially) untracked files to run `git add` internally.